activemount.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/docker/go-plugins-helpers/volume"
+)
+
+type ActiveMount struct {
+	UsageCount int
+}
+
+func (d *DockerOnTop) Activate(request *volume.MountRequest, activemountsdir lockedFile) (bool, error) {
```

Comment:
A comment on both `DockerOnTop.Activate` and `DockerOnTop.Activate`:

1.    Their names probably don't quite reflect what they do. If I saw the following piece of code:
      ```go
      var d DockerOnTop = ...
      d.Activate(...)
      ```
      I would most probably assume that `d.Activate` _activates_ the `d` object (whatever it could mean) because that's what the name suggests. But what it actually does is it activates _a volume_. If we had a proper volume object, it would make sense to move this method there and have `volume.Activate`, but as its a method of `DockerOnTop`, its name should probably be `ActivateVolume`

2.  In go, the case of the first letter (i.e., whether it's lowercase or uppercase) has a semantic effect: if an entity name is capitalized, it's _exported_ (i.e., public - available from other packages), otherwise it's _unexported_ (i.e., private, only available in this package). As I understand, the `.Activate` and `.Deactivate` are both intended for internal use only (a caller external to `DockerOnTop` shall not call them directly). If that's the case, their names should be changed.

3.  If I'm wrong and you intended that these functions are exported, it's a go convention to have a documentation string for every exported entity (see [Documenting go code](https://go.dev/blog/godoc) for details). You may notice that other exported methods of `DockerOnTop` don't have documentation, which is a flaw, but all of them correspond to Docker API calls, whereas the two new functions you are adding are something new.

4.  Finally, as you may notice in further comments, at some points it wasn't clear to me what these functions should do and, especially, what they should return. That's why, even if you didn't mean to export them and you'll rename them to `.activateVolume` and `.deactivateVolume` respectively, they would still benefit from a short docstring that explains what they do and what they return.

Reply:
I hope this looks better now with the changes I have done:
Commit: b18043e55f229afdeb0243239d7f25d55d012173

Reply:
Also commit 3a67652e35100e08b72481ca830ffc1c62d54e19 since I didn't properly rename `deactivateVolume`